Factory Pattern/src/OnlineOrderProcessingFactory.java

```diff
@@ -0,0 +1,13 @@
+public class OnlineOrderProcessingFactory {
```

Comment:
if u have new orderType u have to add a new if condition which leads to changing this Factory. Think of a way to avoid it.